test/subdataframe.jl

```diff
@@ -1,6 +1,14 @@
 module TestSubDataFrame
     using Test, DataFrames
 
+    @testset "copy - SubDataFrame" begin
+        df = DataFrame(x = 1:10, y = 1.0:10.0)
+        sdf = view(df, 1:2, 1:1)
+        @test sdf isa SubDataFrame
+        @test copy(sdf) isa DataFrame
+        @test isequal(sdf, copy(sdf))
```

Comment:
`==` is enough.

Reply:
But is is a problem? `isequal` is stricter and handles `missing` (of course in this case it makes no difference, but I want to understand the preferred style).

Reply:
That's just a matter of style. In general I prefer `==` since it's more readable and `isequal` makes it sound like there's something special going on. Also in most files we used `const ≅= isequal`.